src/components/core.ts

```diff
@@ -190,16 +190,23 @@ export default class Core {
     /**
      * Check if holderId is not empty
      */
-    if (!this.config.holderId) {
-      throw Error('«holderId» param must being not empty');
+    if (!this.config.holderId && this.config.holder) {
+      throw Error('«holderId» or «holder» param must being not empty');
     }
 
     /**
      * Check for a holder element's existence
      */
-    if (!$.get(this.config.holderId)) {
+    if (!this.config.holder &&
+        typeof this.config.holderId === 'string' &&
+        !$.get(this.config.holderId)
+      ) {
       throw Error(`element with ID «${this.config.holderId}» is missing. Pass correct holder's ID.`);
     }
+
+    if (this.config.holder && !$.isElement(this.config.holder)) {
```

Comment:
I think `holder` property should also accept string because we will remove `holderId` on major release

Reply:
Reference is not always the most convenient way I guess

Reply:
One thing we do it's call getElementById().
```js
new EditorJS({
 holder: document.getElementById('id'),
})

new EditorJS(document.getElementById('id'))
```
Or user write on 23 symbols more or we add 2 if's

Reply:
Better if it is easier for user

Reply:
update it